### PR TITLE
Add support for running source actions

### PR DESCRIPTION
**Pull request by rchl**
Expose a list of source actions that are provided by the servers.
Servers announce support for those through the
"codeActionProvider.codeActionKinds" capability.

### CODE REVIEW

**Comment by rchl** [comment]
Partially resolves #1356. That issue also asks for being able to run specific code actions from a shortcut/pallette command.

File: Context.sublime-menu [hunk 1]
```diff
@@ -29,7 +29,12 @@
             },
             {
                 "command": "lsp_code_actions",
-                "caption": "Apply Code Action…"
+                "caption": "Code Action…"
```
Comment (rchl) [inline Context.sublime-menu:32]: Changed caption here as this command doesn't apply right away, it requires further input and then applies.

Comment (rwols) [reply Context.sublime-menu:32]: It should also be changed in Main.sublime-mennu

Comment (rwols) [reply Context.sublime-menu:32]: Never mind, it's not in the main menu

**Review by rwols — APPROVED** [review]

**Pull request merged**